core/controllers/reader.py

```diff
@@ -1011,3 +1011,34 @@ def post(self, exploration_id):
             self.payload.get('report_text'),
             self.user_id)
         self.render_json(self.values)
+
+
+class QuestionPlayerHandler(base.BaseHandler):
+    """Provides questions with given skill ids."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    @acl_decorators.open_access
+    def get(self):
+        """Handles GET request."""
+        start_cursor = self.request.get('start_cursor')
+        skill_ids = self.request.get('skill_ids').split(',')
```

Comment:
Add comment explaining that the skill IDs are given as a comma-separated list because this is a GET request.

Reply:
👍 done